src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -263,13 +265,25 @@ def acr_queue(cmd,
         result = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
             build_request=build_request, resource_group_name=resource_group_name, registry_name=registry_name))
 
-        print("Queued a build with build-id: {}.".format(result.build_id))
-
         if no_logs == False:
-            print("Starting to stream the logs...")
+            size = os.path.getsize(FILE)
+            unit = ""
+            for S in ['Bytes', 'KB', 'MB']:
```

Comment:
Not that it really matters but it fails for other cases like GB/TB, may want to just normalize everything to be MB and check against that, specific unit doesn't really matter - it's just flavor text for the log